File: 백준/Silver/7568. 덩치/덩치.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n, rank;
    cin >> n;
    vector<vector<int>> arr(n, vector<int>(2, 0));
    for (int i = 0; i < n; i++)
        cin >> arr[i][0] >> arr[i][1];
    for (int i = 0; i < n; i++) {
        rank = 1;
        for (int j = 0; j < n; j++) {
            if (arr[i][0] < arr[j][0] && arr[i][1] < arr[j][1])
                rank += 1;
        }
        cout << rank << " ";
    }
    return 0;
}

File: 백준/Silver/7568. 덩치/덩치.py
import sys
input=lambda:sys.stdin.readline().rstrip()
arr=[[*map(int,input().split())] for _ in range(int(input()))]
for a in arr:
    rank=1
    for b in arr:
        if a[0]<b[0] and a[1]<b[1]:
            rank+=1
    print(rank,end=" ")